api-specificatie/openapi.yaml

```diff
@@ -447,8 +401,6 @@ components:
           $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/hallinks.yaml#/HalLink'
         # De standaard relatie richting is van openbare ruimte naar woonplaats. Voor andersom is een hulp link nodig.
         openbareruimten:
-          # JBO: Een description op deze manier naast een $ref plaatsen mag niet binnen OAS3. Het geeft geen error, maar wordt ook niet getoond in de swagger weergave en is dan dus nutteloos. 
-          # JBO: Dit is op te lossen door het toepassen van een allOf. 
           allOf: 
             - description: 'De gerelateerde openbare ruimte op basis waarvan het adres is samengesteld.'
             - $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/hallinks.yaml#/HalLink'
```

Comment:
Moet hier niet een array van openbare ruimte ID's worden opgenomen i.p.v. links? De links komen toch in de Adres_links  te staan? Dit geldt ook voor de andere gerelateerde objecten.

Reply:
Deze opmerking staat bij de adres_Links. Ik weet niet zo goed wat je dan precies bedoelt. Wil je dat er een array van id's in het adres zelf opgenomen wordt ? 
Gaarne een issue inschieten, Deze pull  request wil ik vandaag mergen.